src/geointerface.jl

```diff
@@ -1,2 +1,14 @@
 GI.extent(ga::GeoArray) = bbox(ga)
 GI.crs(ga::GeoArray) = isempty(GFT.val(crs(ga))) ? nothing : crs(ga)
+GI.israster(::Type{GeoArray}) = true
+GI.crstrait(ga::GeoArray) = _crstrait(GI.crs(ga))
+
+_crstrait(::Nothing) = GI.UnknownTrait()
+function _crstrait(crs)
+    acrs = ArchGDAL.importCRS(crs)
+    Bool(ArchGDAL.GDAL.osrisgeographic(acrs.ptr)) && return GI.GeographicTrait()
+    Bool(ArchGDAL.GDAL.osrisprojected(acrs.ptr)) && return GI.ProjectedTrait()
+    @error "Unknown CRS type, please report this issue for the given crs/file"
```

Comment:
We probably also need a global LocalCRS trait

Reply:
We can mirror all Proj PROJTYPE as traits, but I think a Local one is assumed to be projected (as in metres) for our purposes here?

Reply:
Maybe that can be a subtype of AbstractProjected?  Or AbstractProjected is a subtype of AbstractCartesian